packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -572,7 +581,7 @@ const VirtualListBaseFactory = (type) => {
 							// So we find nextIndex only when currentExtent is different from candidateExtent.
 							if (currentExtent !== candidateExtent) {
 								nextIndex = this.findNearestSpottableItemInExtent(currentIndex, candidateExtent);
-								animate = (wrap === true);
+								animate = this.props.animate && (wrap === true);
```

Comment:
Not sure what this line means. But I added checking `animate` to disable it anyway.